boosters/node-beginner/tests/07_exercise.test.js

```diff
@@ -0,0 +1,77 @@
+const request = require("supertest");
+const app = require("../src/server");
+const teardown = require("./config/teardown");
+const uuid = require("uuid/v4");
+const setup = require("./config/setup");
+
+const users = [
+  { firstName: "Henry", lastName: "Chao", id: uuid() },
+  { firstName: "Andrew", lastName: "Mayer", id: uuid() },
+  { firstName: "Liang", lastName: "Chen", id: uuid() },
+];
+
+const repos = users.reduce(
+  (acc, curr) => [
+    ...acc,
+    { name: "repo1", userId: curr.id, isPrivate: false, id: uuid() },
+    { name: "repo2", userId: curr.id, isPrivate: true, id: uuid() },
+  ],
+  []
+);
+
+describe("Not sure yet", () => {
+  beforeAll(async () => {
+    await setup("Users", users)();
+    return setup("Repos", repos)();
+  });
+
+  afterAll(teardown);
+
+  xit("Should return all public repositories and all repositories of current user", async () => {
```

Comment:
Pick one user to be the current user, then validate that all public and owned repositories are returned.